frontend/src/components/IssueV1/components/Sidebar/VCSInfo.vue

```diff
@@ -1,12 +1,60 @@
 <template>
   <div
-    v-if="false"
-    class="text-sm text-control-light flex flex-col items-start gap-y-1"
-  ></div>
+    v-if="issue.planEntity?.vcsSource"
+    class="text-sm text-control-light flex space-x-1 items-center"
+  >
+    <VCSIcon v-if="vcsProvider" :type="vcsProvider.type" />
+    <a
```

Comment:
Btw, also consider vcsConnector is not present either
1) the issue is created via API without VCS connector set.
2) IAM permission issue that cannot get the VCS connector or provider info.

Reply:
LGTM, add checks for permission